hail/src/main/scala/is/hail/asm4s/Code.scala

```diff
@@ -859,23 +859,33 @@ object FieldRef {
   }
 }
 
-trait Settable[T] {
-  def load(): Code[T]
+object Value {
+  def apply[T](c: Value[_], insn: => AbstractInsnNode): Value[T] =
```

Comment:
Could you explain this? It isn't used in this PR, and I don't understand the intended use.

Reply:
Ah, sorry, that's left over from a previous attempt.  Will delete.